src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.cs

```diff
@@ -91,6 +91,8 @@ public FieldBuilder DefineLiteral(string literalName, object? literalValue)
 
         public override Type? BaseType => m_typeBuilder.BaseType;
 
+        public override bool IsByRefLike => false;
```

Comment:
Is it the case that all of these throw NotSupportedException today and this PR is just making them return false instead?

Reply:
That's correct:

```
System.Reflection.Emit.Tests.EnumBuilderPropertyTests.IsByRefLikeFalse [FAIL]
        System.NotSupportedException : Derived classes must provide an implementation.
        Stack Trace:
          /dotnet/runtime/src/libraries/System.Private.CoreLib/src/System/Type.cs(54,0): at System.Type.get_IsByRefLike()
          /dotnet/runtime/src/libraries/System.Reflection.Emit/tests/EnumBuilder/EnumBuilder.Properties.Tests.cs(63,0): at System.Reflection.Emit.Tests.EnumBuilderPropertyTests.IsByRefLikeFalse()
```

Sorry, I forgot tests; adding them now.